ProteinCartography/esmfold_apiquery.py

```diff
@@ -123,7 +124,12 @@ def main():
 
     input_file = args.input
     output_file = args.output
-    esmfold_apiquery(input_file, output_file)
+
+    #Ignore warnings when we make the ESMFold API request
+    #We're disabling SSL which `requests` will otherwise warn us about
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
```

Comment:
From my understanding, this would ignore all warnings that occur within this block. 

If we're trying just to ignore SSL warnings, should we be more specific and use `urllib3.exceptions.InsecureRequestWarning` and/or `ssl.SSLErrorWarning`?

Ignoring these kinds of warnings also seems like it might be a security vulnerability.

@mertcelebi what do you think?

Reply:
Yeah, this should explicitly ignore the `InsecureRequestWarning` and that should work. We still would like to surface other warnings to the user. 

If you import, `from requests.packages.urllib3.exceptions import InsecureRequestWarning` and change this line to `warnings.simplefilter("ignore", category=InsecureRequestWarning)`, that would work.